scalability_and_performance/optimizing-networking.adoc

```diff
@@ -0,0 +1,36 @@
+[id="optimizing-networking"]
+= Optimizing networking
+include::modules/common-attributes.adoc[]
+:context: optimizing-networking
+
+toc::[]
+
+The xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShift SDN] uses OpenvSwitch, virtual extensible LAN (VXLAN) tunnels, OpenFlow rules, and iptables. This network can be tuned by using jumbo frames, network interface cards (NIC) offloads, multi-queue, and ethtool settings.
+
+OVN-Kubernetes uses Geneve (Generic Network Virtualization Encapsulation) instead of VXLAN as the tunnel protocol.
```

Comment:
And it's possible to xref to OVN-Kubernetes as well.

`xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes`